documentation/common/attributes.adoc

```diff
@@ -0,0 +1,38 @@
+// AsciiDoc settings
+:data-uri!:
+:doctype: book
+:experimental:
+:idprefix:
+:imagesdir: images
+:numbered:
+:sectanchors!:
+:sectnums:
+:source-highlighter: highlightjs
+:toc: left
+:linkattrs:
+:toclevels: 4
+
+// Name placeholders
+:ProductLongName: Strimzi
+:ProductName: Strimzi
+:ProductVersion: master
+:OpenShiftName: OpenShift
+:OpenShiftLongName: OpenShift
+:KubernetesName: Kubernetes
+:KubernetesLongName: Kubernetes
+:ProductPlatformName: {OpenShiftName} or {KubernetesName}
+:ProductPlatformLongName: {OpenShiftLongName} or {KubernetesLongName}
+
+// Docker image names
+:DockerRepository: https://hub.docker.com[Docker Hub]
+:DockerZookeeper: strimzi/zookeeper:{ProductVersion}
+:DockerKafka: strimzi/kafka:{ProductVersion}
+:DockerKafkaConnect: strimzi/kafka-connect:{ProductVersion}
+:DockerKafkaConnectS2I: strimzi/kafka-connect-s2i:{ProductVersion}
+:DockerTopicController: strimzi/topic-controller:{ProductVersion}
```

Comment:
having {ProductVersion} with "master" value means something like strimzi/kafka:master but it's not a tag we are using right ? We should have strimzi/kafka:latest. Maybe we need a different placeholder for the Docker image tag ?

Reply:
Hmm, this is good point. This works for release but might not work for master. Or it might because it actually uses latest as version for the docker images. I will give it a try.

Reply:
Hmm, this actually works fine in master. The `RELEASE_VERSION` which is used for this is set to latest in master.

IMHO this is a bit confusing, since the Java bits now have version `0.5.0-SNAPSHOT` ... maybe we should use this instead for the latest for everything. But that is far beyond the scope of this.

Reply:
I anyway improved this, so although the Docker Tag is still set by default to the version, it can be now configured separately.

Reply:
+1